### PR TITLE
Add system-color

### PR DESCRIPTION
**Pull request by jamesnw**
https://github.com/Fyrd/caniuse/issues/4179 shows interest as a feature, including links to posts about it.

### CODE REVIEW

File: features/system-color.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: system-color
```
Comment (ddbeck) [inline features/system-color.yml:1]: This seems to be a bit more "natural" in use than the hyphenated form. Also, see my commentary on the description:

```suggestion
name: System colors
```

File: features/system-color.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+description: The `system-color` CSS keywords allow authors to match default colors from the user agent.
```
Comment (ddbeck) [inline features/system-color.yml:2]: As written, this suggests `system-color` is a keyword you can actually type and get something from. I think that's not the case. I'd suggest something along these lines, using examples to represent the real-world use:

```suggestion
description: The system color CSS keywords, such as `ButtonBorder` or `LinkText`,  allow authors to match default colors from the user agent.
```

File: features/system-color.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+spec: https://drafts.csswg.org/css-color-4/#css-system-colors
+group: css
+status:
+  compute_from: css.types.color.system-color
+compat_features:
+  - css.types.color.system-color
```
Comment (ddbeck) [inline features/system-color.yml:8]: This is not a problem here, but CSS types in browser compat data (BCD) are quite messy. We'll probably see some cases where we'll have to write support statuses by hand, or have no status at all. I was about to suggest having a note about this, but I think feature grouping will probably do the work here.

**Review by ddbeck — CHANGES_REQUESTED** [review]

**Comment by jamesnw** [comment]
Replaced by #1619

**Pull request closed without merging**